src/app/CommandSender.h

```diff
@@ -507,8 +509,10 @@ class CommandSender final : public Messaging::ExchangeDelegate
     TLV::TLVType mDataElementContainerType = TLV::kTLVType_NotSpecified;
 
     chip::System::PacketBufferTLVWriter mCommandMessageWriter;
-    uint16_t mFinishedCommandCount    = 0;
-    uint16_t mRemoteMaxPathsPerInvoke = 1;
+
+    uint16_t mInvokeResponseMessageCount = 0;
```

Comment:
Are the size automation numbers just confused, or is this a significant regression for some reason?

Reply:
I am merging master in this PR to see is that helps as I am not entirely sure how the comparison is done and if it is doing a checkout based on commit ID directly without merging master to to a more accurate comparison

Reply:
After merging master the delta is much smaller. See https://github.com/project-chip/connectedhomeip/pull/31781#issuecomment-1927736359